src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,83 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private Entry<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Entry[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        if (size >= LOAD_FACTOR * table.length) {
+            resize();
+        }
```

Comment:
not a mistake, but if resize is only ever called inside of `if`, you can move `if` inside of resize